### PR TITLE
Add CODEOWNERS file

### PR DESCRIPTION
**Pull request by lindluni**
Add CODEOWNERS file to release-1.4 branch

Signed-off-by: Brett Logan <Brett.T.Logan@ibm.com>

#### Type of change

- Documentation update

### CODE REVIEW

**Comment by lindluni** [comment]
Need 2 approvals and then @mastersingh24 or @denyeart  can force merge as Admins since we can't get CODEOWNERS approval

**Review by mastersingh24 — APPROVED** [review]

**Pull request merged**
